src/catalog.js

```diff
@@ -122,6 +122,14 @@ angular.module('gettext').factory('gettextCatalog', function (gettextPlurals, ge
         currentLanguage: 'en',
         /**
          * @ngdoc property
+         * @name gettextCatalog#localeSeparator
+         * @public
+         * @type {String}
+         * @description Language code locale separator. As example "en-US".
+         */
+        localeSeparator: '-',
```

Comment:
Adding this will cause the default to be `-`, which would be a compatibility break, since we currently have `_`.